Formula/fcp.rb

```diff
@@ -0,0 +1,30 @@
+class Fcp < Formula
+  desc "Significantly faster alternative to the classic Unix cp(1) command"
+  homepage "https://github.com/Svetlitski/fcp/"
+  url "https://github.com/Svetlitski/fcp/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "7e4e69bdb7f1f831dc52cd517afdd5722313d3dece3afd7dad418d224d4badd1"
+  license "BSD-3-Clause"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"src.old").write "Hello world!"
+    system "#{bin}/fcp", "src.old", "dest.txt"
```

Comment:
```suggestion
    system bin/"fcp", "src.old", "dest.txt"
```
Minor style suggestion.